### DISCUSSION TITLE
webpack dist folder structure

### DISCUSSION THREAD

**Discussion by gaby64** [quasarframework/quasar#10649] [General - Components / Directives / etc]
im wondering why this happens:
`...\dist\views\contracts~views\documents~views\equipment~views\reports~views\upload.js`

its the only file in that strange directory structure, the contents is QUploader

any way to avoid having it in a directory like that?

im using vue-cli v4 and quasar v2

**Comment by cyfung1031** [comment]
If you don't like the webpack feature, you can use vite2. It is faster than vue-cli v4.

https://vitejs.dev/

**Comment by gaby64** [comment]
I would not know how to port over my config to vite, I use a php backend with wamp
